gnome/virt-manager/Portfile

```diff
@@ -9,6 +9,7 @@ categories          gnome emulators
 supported_archs     noarch
 maintainers         {danchr @danchr} openmaintainer
 license             GPL-2+
+revision            1
```

Comment:
a change to the description does not require a `revision` increase as the installed files do not change

Reply:
Fixed.